tests/Fixture/Entity/EntityForValidation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\Tests\Fixture\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Zenstruck\Foundry\Tests\Fixture\Model\Base;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ORM\Entity]
+class EntityForValidation extends Base
+{
+    public function __construct(
+        #[ORM\Column()]
+        #[Assert\NotBlank()]
```

Comment:
Hello @nikophil, what about validation groups?

Reply:
hi @COil  :wave: 

yeah good call, I'll add them as a parameter of `withValidation()` method

Reply:
done